docs/data/styles/advanced/advanced-pt.md

```diff
@@ -444,7 +444,7 @@ const className = `${sheetName}-${ruleName}-${identifier}`;
 const productionPrefix = 'jss';
 const identifier = 123;
 
-const className = `${productionPrefix}-${identifier}`;
+const className = `${productionPrefix}${identifier}`;
```

Comment:
Note that we are not manually changing the translation files, please revert the changes to the `advanced-pt.md` and the `advanced-zh.md` files.

Reply:
👍 

Done.